_kasu_smoke.py

```
@@ -0,0 +1,14 @@
+"""Smoke test for the Kasu PR reviewer. DO NOT MERGE - safe to delete."""
+
+
+def average_price(items):
+    """Average unit price across cart line items."""
+    total = 0
+    for it in items:
+        total += it["price"]
+    return total / len(items)  # bug: ZeroDivisionError when items is empty
+
+
+def last_item(items):
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `last_item`

(ANN202)